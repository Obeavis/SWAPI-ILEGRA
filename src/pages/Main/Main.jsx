import React from "react";
import RootSelect from "../../containers/RootSelect/RootSelect"

const Main = () => (
	<div className="main my-5">
		<RootSelect />
	</div>
)

export default Main;