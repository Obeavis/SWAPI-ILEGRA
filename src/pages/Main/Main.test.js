import React from "react";
import { MemoryRouter } from 'react-router-dom';
import Adapter from "enzyme-adapter-react-16";
import renderer from "react-test-renderer";
import { configure, mount } from "enzyme";
import Main from "./Main";

configure({ adapter: new Adapter() });

describe("Film Component", () => {

    const component = (
        <MemoryRouter>
            <Main />
        </MemoryRouter>
    );

    it("matches the snapshot", () => {
        const tree = renderer.create(component).toJSON();
        
        expect(tree).toMatchSnapshot();
    });

    it('Rendering', () => {
        const wrapper = mount(component)
        expect(wrapper.find('RootSelect').exists()).toEqual(true)
    });

})