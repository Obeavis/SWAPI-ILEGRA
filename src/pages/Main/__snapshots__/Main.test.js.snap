// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Film Component matches the snapshot 1`] = `
<div
  className="main my-5"
>
  <div
    className="root-select container"
  >
    <div
      className="justify-content-center row"
    >
      <div
        className="mb-4 col-xl-4 col-lg-6 col-11"
        onClick={[Function]}
      >
        <div
          className="box"
        >
          <img
            alt="Films"
            className="img-fluid"
            src="all-films.jpg"
          />
          <span
            className="span-box"
          >
            <span
              className="thumb-title text-bold text-center"
            >
              Films
            </span>
          </span>
        </div>
      </div>
      <div
        className="mb-4 col-xl-4 col-lg-6 col-11"
      >
        <div
          className="box"
        >
          <img
            alt="People"
            className="img-fluid"
            src="all-people.jpg"
          />
          <span
            className="span-box"
          >
            <span
              className="thumb-title text-bold text-center"
            >
              People
            </span>
          </span>
        </div>
      </div>
      <div
        className="mb-4 col-xl-4 col-lg-6 col-11"
      >
        <div
          className="box"
        >
          <img
            alt=""
            className="img-fluid"
            src="all-starships.jpg"
          />
          <span
            className="span-box"
          >
            <span
              className="thumb-title text-bold text-center"
            >
              Starships
            </span>
          </span>
        </div>
      </div>
    </div>
    <div
      className="justify-content-center row"
    >
      <div
        className="mb-4 col-xl-4 col-lg-6 col-11"
      >
        <div
          className="box"
        >
          <img
            alt=""
            className="img-fluid"
            src="all-vehicles.jpg"
          />
          <span
            className="span-box"
          >
            <span
              className="thumb-title text-bold text-center"
            >
              Vehicles
            </span>
          </span>
        </div>
      </div>
      <div
        className="mb-4 col-xl-4 col-lg-6 col-11"
      >
        <div
          className="box"
        >
          <img
            alt=""
            className="img-fluid"
            src="all-species.jpg"
          />
          <span
            className="span-box"
          >
            <span
              className="thumb-title text-bold text-center"
            >
              Species
            </span>
          </span>
        </div>
      </div>
      <div
        className="mb-4 col-xl-4 col-lg-6 col-11"
      >
        <div
          className="box"
        >
          <img
            alt=""
            className="img-fluid"
            src="all-planets.jpg"
          />
          <span
            className="span-box"
          >
            <span
              className="thumb-title text-bold text-center"
            >
              Planets
            </span>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
